dataactbroker/handlers/accountHandler.py

```diff
@@ -646,9 +604,9 @@ def list_users_with_status(self):
 
         try:
             if self.interfaces.userDb.hasPermission(current_user, "agency_admin"):
-                users = self.interfaces.userDb.getUsersByStatus(request_dict.getValue("status"), current_user.cgac_code)
+                users = sess.query(User).filter(User.user_status_id == USER_STATUS_DICT[request_dict.getValue("status")], User.cgac_code == current_user.cgac_code).all()
```

Comment:
A minor, optional thing, but I've been loving the `filter_by` syntactic sugar that CM turned me on to:
```python
users = sess.query(User).filter_by(
    user_status_id = USER_STATUS_DICT[request_dict.getValue('status')],
    cgac_code = current_user.cgac_code).\
    all()
```